test/integration/targets/eos_interfaces/tests/cli/replaced.yaml

```
@@ -25,6 +27,20 @@
 - assert:
     that:
       - "ansible_facts.network_resources.interfaces|symmetric_difference(result.before)|length == 0"
+      - result is changed
+
+- name: Replaces device configuration of listed interfaces with provided configuration
+  eos_interfaces:
+    config: "{{ config }}"
+    state: replaced
+  register: result
+  become: yes
+
+- assert:
+    that:
+      - "ansible_facts.network_resources.interfaces|symmetric_difference(result.before)|length == 0"
```

```suggestion
      - "ansible_facts.network_resources.interfaces|symmetric_difference(result.before) == []"
```

It doesn't have to be changed everywhere, but I prefer to use this when I see it as it is a tiny bit more explicit.